### DISCUSSION TITLE
Microsoft company account Enter two-factor authentication code 6 dig is not correct, can't find recover code

### DISCUSSION THREAD

**Discussion by yewanxuan** [community/community#39716] [Repositories]
### Select Topic Area

Product Feedback

### Body

into the Account recovery page, three option function are all disable to select. Please help

related account : wanxuanye@microsoft.com

**Comment by ossanna16** [comment]
Hi there! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
